services/vision/vision.go

```diff
@@ -118,10 +119,21 @@ func FindFirstName(r robot.Robot) string {
 	return ""
 }
 
-// FirstFromRobot returns the first vision service in this robot.
+// FirstFromRobot returns the first vision service on this robot.
 func FirstFromRobot(r robot.Robot) (Service, error) {
-	name := FindFirstName(r)
-	return FromRobot(r, name)
+	return FirstFromLocalRobot(r)
```

Comment:
I would still return any vision service if FirstFromLocalRobot fails. Or just remove this function entirely haha

Reply:
Okay valid and done. Look again